src/patternfly/components/Button/docs/design.md

```diff
@@ -1,4 +1,59 @@
-This is a placeholder for design/usage documentation for this component. Should consider:
-* Usage guidelines
-* Visual styling considerations.
-* Guidance on content and labeling of buttons.
+# Buttons
+Buttons communicate and trigger actions a user can take throughout an experience.
+
+
+## Usage
+***
+A button should represent an action a user can take, like submitting a form, cancelling a process, or creating a new object. When the user clicks the button, the associated action should be performed immediately.
+
+### Button Types
+Use different button types to build hierarchy in an application layout.
+
+![Buttons Grid](img/buttons.png)
+
+#### Primary
+Use primary buttons for the main action you want a user to take on a page. In general, try to limit primary buttons to one per page.
+
+#### Secondary
+Use secondary buttons for secondary actions a user can take. For example, a cancel button would be secondary to a submit button on a form or modal.
+
+#### Tertiary
+Use tertiary buttons for any action a user can take on a page that is not primary or secondary. For example, actions a user might take to edit items in a list could be represented as tertiary buttons.
+
+#### Danger
```

Comment:
In the image these are labeled as "Destructive" buttons, which is consistent with how we've named these before. Are you recommending we change that @stacymcauliffe ?